guacamole/src/main/webapp/translations/de.json

```diff
@@ -4,103 +4,117 @@
+        "ERROR_UPLOAD_204"     : "Das Ziel für die Dateiübertragung existiert nicht. Bitte überprüfen Sie, ob das Ziel vorhanden ist, und versuchen Sie es erneut.",
+        "ERROR_UPLOAD_205"     : "Das Ziel für die Dateiübertragung ist derzeit gesperrt. Warten Sie, bis alle laufenden Aufgaben abgeschlossen sind, und versuchen Sie es erneut.",
+        "ERROR_UPLOAD_301"     : "Sie haben keine Berechtigung diese Datei hochzuladen, da Sie nicht angemeldet sind. Melden Sie sich an und versuchen Sie es erneut.",
+        "ERROR_UPLOAD_303"     : "Sie haben keine Berechtigung, diese Datei hochzuladen. Wenn Sie Zugriff benötigen, überprüfe die Systemeinstellungen oder wenden Sie sich an Ihren Systemadministrator.",
+        "ERROR_UPLOAD_308"     : "Die Dateiübertragung weist keinen Fortschritt auf. Dies wird häufig durch Netzwerkprobleme verursacht, wie z.B. schlechtes WLAN-Signal oder  eine sehr langsame Netzwerkverbindung. Bitte überprüfen Sie das Netzwerk und versuchen Sie es erneut.",
+        "ERROR_UPLOAD_31D"     : "Derzeit werden zu viele Dateien übertragen. Warten Sie, bis die vorhandenen Übertragungen abgeschlossen sind, und versuchen Sie es dann erneut.",
+        "ERROR_UPLOAD_DEFAULT" : "Auf dem Guacamole-Server ist ein interner Fehler aufgetreten und die Verbindung wurde beendet. Wenn das Problem weiterhin besteht, benachrichtigen Sie bitte Ihren Systemadministrator oder überprüfen Sie Ihre Systemprotokolle.",
+
+        "HELP_CLIPBOARD"           : "Kopierter / ausgeschnittener Text in Guacamole wird hier angezeigt. Änderungen am folgenden Text wirken sich auf die entfernte Zwischenablage aus.",
+        "HELP_INPUT_METHOD_NONE"   : "Es wird keine Eingabemethode verwendet. Tastatureingaben werden von einer angeschlossenen, physischen Tastatur akzeptiert.",
+        "HELP_INPUT_METHOD_OSK"    : "Anzeigen und Akzeptieren von Eingaben über die eingebaute Guacamole-Bildschirmtastatur. Die Bildschirmtastatur ermöglicht die Eingabe von Tastenkombinationen, die ansonsten möglicherweise nicht möglich sind (z. B. Strg-Alt-Entf).",
+        "HELP_INPUT_METHOD_TEXT"   : "Ermöglicht die Eingabe von Text und emuliert Tastaturereignisse basierend auf dem eingegebenen Text. Dies ist für Geräte wie Mobiltelefone erforderlich, denen eine physische Tastatur fehlt.",
+        "HELP_MOUSE_MODE"          : "Legt fest, wie sich die Remote-Maus in Bezug auf Berührungen verhält.",
+        "HELP_MOUSE_MODE_ABSOLUTE" : "Tippe um zu klicken. Das Klicken erfolgt am Ort der Berührung.",
+        "HELP_MOUSE_MODE_RELATIVE" : "Ziehen Sie, um den Mauszeiger zu bewegen und tippen Sie, um zu klicken. Der Klick erfolgt an der Position des Mauszeigers.",
+        "HELP_SHARE_LINK"          : "Die aktuelle Verbindung wird gemeinsam genutzt und kann von jedem mit den folgenden {LINKS, Plural, einem {Link} anderen {Links} aufgerufen werden:",
```

Comment:
I don't believe this will work, as it's not correct ICU format for pluralization. Shouldn't this be `... mit {LINKS, plural, one{foldendem Link} other{den folgenden Links}} ...`?